vendor/assets/javascripts/behaviors/changed_form_modal.js

```diff
@@ -1,6 +1,6 @@
 var NR = Cmsify.NestedResource.prototype;
 var CFM = Cmsify.ChangedFormModal = function(el) {
-  this.$el = $('form');
+  this.$el = $('form.js-track-changes');
```

Comment:
This should be handled with the initializer. I'm not sure why `this.$el` isn't being set to `$(el)`, but we should only initialize this class for forms that need it.